docs/integrations/sources/google-search-console.md

```diff
@@ -1,95 +1,83 @@
 # Google Search Console
 
-This page contains the setup guide and reference information for the google search console source connector.
+This page contains the setup guide and reference information for the Google Search Console source connector.
 
 ## Prerequisites
 
 - A verified property in Google Search Console
-- Enable Google Search Console API for GCP project at [GCP console](https://console.cloud.google.com/apis/library/searchconsole.googleapis.com)
-- Credentials to a Google Service Account \(or Google Service Account with delegated Domain Wide Authority\) or Google User Account
-- Enable Google Search Console API
+- Google Search Console API enabled for your project
 
 ## Setup guide
 
-### Step 1: Set up google search console
+### Step 1: Set up Google Search Console authentication
 
-#### How to create the client credentials for Google Search Console, to use with Airbyte?
+To authenticate the Google Search Console connector, you will need to use one of the following methods:
 
-You can either:
+#### I: OAuth (Recommended for Airbyte Cloud)
 
-- Use the existing `Service Account` for your Google Project with granted Admin Permissions
-- Use your personal Google User Account with oauth. If you choose this option, your account must have permissions to view the Google Search Console project you choose.
-- Create the new `Service Account` credentials for your Google Project, and grant Admin Permissions to it
-- Follow the `Delegating domain-wide authority` process to obtain the necessary permissions to your google account from the administrator of Workspace
+You can authenticate using your Google Account with OAuth if you are the owner of the Google Search Console property or have view permissions. Follow [Google's instructions](https://support.google.com/webmasters/answer/7687615?sjid=11103698321670173176-NA) to ensure that your account has the necessary permissions (**Owner** or **Full User**) to view the Google Search Console property. This option is recommended for **Airbyte Cloud** users, as it significantly simplifies the setup process and allows you to authenticate the connection [directly from the Airbyte UI](#step-2-set-up-the-google-search-console-connector-in-airbyte).
 
-### Creating a Google service account
+#### II: Google service account with JSON key file (Recommended for Airbyte Open Source)
```

Comment:
can we hide this for cloud?

Reply:
I've gone ahead and added the tags just in case, but I don't think they have any effect on the main page when an in-app page exists, since the latter is the one that gets displayed in the Airbyte UI